cosmwasm/tools/deploy-pyth-bridge.js

```diff
@@ -64,38 +61,96 @@ const artifact = argv.artifact;
 const CONFIG = {
   mainnet: {
     terraHost: {
-      URL: "https://lcd.terra.dev",
-      chainID: "columbus-5",
+      URL: "https://phoenix-lcd.terra.dev",
+      chainID: "phoenix-1",
       name: "mainnet",
     },
-    wormholeContract: "terra1dq03ugtd40zu9hcgdzrsq6z2z4hwhc9tqk2uy5",
-    pythEmitterAddress:
-      "6bb14509a612f01fbbc4cffeebd4bbfb492a86df717ebe92eb6df432a3f00a25",
+    pyth_config: {
+      wormhole_contract:
+        "terra12mrnzvhx3rpej6843uge2yyfppfyd3u9c3uq223q8sl48huz9juqffcnh",
+      data_sources: [
+        {
+          emitter: Buffer.from(
+            "6bb14509a612f01fbbc4cffeebd4bbfb492a86df717ebe92eb6df432a3f00a25",
+            "hex"
+          ).toString("base64"),
+          chain_id: 1,
+        },
+        {
+          emitter: Buffer.from(
+            "f8cd23c2ab91237730770bbea08d61005cdda0984348f3f6eecb559638c0bba0",
+            "hex"
+          ).toString("base64"),
+          chain_id: 26,
+        },
+      ],
+      governance_source: {
+        emitter: Buffer.from(
+          "5635979a221c34931e32620b9293a463065555ea71fe97cd6237ade875b12e9e",
+          "hex"
+        ).toString("base64"),
+        chain_id: 1,
+      },
+      governance_source_index: 0,
+      governance_sequence_number: 0,
+      chain_id: 18,
```

Comment:
not for this pr, can we query this from wormhole contract instead of manually specifying it?

Reply:
I looked into this as part of the cleanup PR before. Unfortunately wormhole doesn't expose a query for it. We could read it out of their config struct, but that felt a little hacky. wdyt?

Reply:
or we could add support to it to the wormhole contract?

Reply:
ok never mind then 😅 second one is good but it takes time for wh to merge and then deploy it. let's stick to this one 😅